typings/index.d.ts

```diff
@@ -0,0 +1,84 @@
+declare module "discord-starboards" {
+    import { EventEmitter } from "events";
+    import * as DJS from "discord.js";
+  
+    export type TranslateClick = (msg: DJS.Message) => void;
+  
+    export interface StarboardOptions {
+      storage?: string | boolean;
+      translateClickHere?: string | TranslateClick;
+    }
+  
+    export interface Starboard {
```

Comment:
Maybe there should be the parameter `manager` ? (I don't know much about types haha)